### DISCUSSION TITLE
How do jou add show episodes with DV.HDR in the title?

### DISCUSSION THREAD

**Discussion by McEnnes** [pymedusa/Medusa#11405] [Q&A]
My Samsung TV does not play Dolby Vision, so i have put DV and DOVI in my ignored words search settings.
In preferred words i have put 2160p and HDR
My Allowed Qualities are 1080P and the Preferred Qualities are 4k
That worked  pretty good so far.
Now there are episodes that have DV.HDR in the name and they do play on my TV, so i do want these when there are no episodes with only 2160p or HDR in the name.
However, Medusa drops these files because of the DV in the name and the result is that  the better quality episodes are not downloaded.
Is there any way to make medusa download DV.HDR files? I have put DV HDR and DV.HDR in my preferred words list, but that does not work. I guess because ignored words is processed first?

**Comment by naturedamends** [comment]
What provider are you using? 
I think in the settings > search settings > episode settings. You can set ignored words, required words etc. I can't get any providers to work.

**Reply by McEnnes** [reply]
I am using multiple providers, Dognzb for one.
I do not think the provider settings will make a difference, since the settings i described are Medusa settings.
The problem is basically that i do not want the episodes that have Only Dolby Vision  (DV or DoVi), but i do want the episodes with HDR or DV.HDR in the name (witch works with Samsung TV`s.

**Reply by naturedamends** [reply]
ignore: DV, DoVi
required: HDR, DV.HDR

??
thanks for response. I'm not sure, i just reading the code.

**Comment by McEnnes** [comment]
I think i solved this.
The DV Or DoVi words i have put in undesired words, not in Ignored words.
By the looks of it, that did the trick
